src/IngredientList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './IngredientList.scss';
+
+function IngredientList({ recipe }) {
+
+    const ingredients = [];
+
+    const ingredientKeys = Object.keys(recipe).filter(k => { 
+        return k.includes('strIngredient') && recipe[k] !== ""
+    });
```

Comment:
so since this is a one line function inside the filter, you can:

```
const ingredientKeys = Object.keys(recipe).filter(k => k.includes('strIngredient') && recipe[k] !== "");
```
implied return statements are popular these days.